npc/jobs/valkyrie.txt

```diff
@@ -217,45 +217,165 @@ yuno_in02,88,164,5	script	Metheus Sylphe#Library	2_M_SAGE_B,{
+								erasequest 5160;
+						}
+						mes "[Metheus Sylphe]";
+						mes "Thank you, your donation will be used for a good cause. You may";
+						mes "now go in and read the book.";
+						close;
+					}
 					mes "[Metheus Sylphe]";
-					mes "Thank you, your donation will be used for a good cause. You may";
-					mes "now go in and read the book.";
+					mes "Unfortunately, you don't seem to possess enough zeny at the moment. Please check your funds and come back again.";
 					close;
-				}
-				mes "[Metheus Sylphe]";
-				mes "Unfortunately, you don't seem to possess enough zeny at the moment. Please check your funds and come back again.";
-				close;
+				case 2:
+					mes "[Metheus Sylphe]";
+					mes "It is to preserve the original 'Book of Ymir,' we have decided to accept donations from people who wish to read the copy we have provided.";
+					next;
+					mes "[Metheus Sylphe]";
+					mes "Although I understand your feeling, but that's the rule.";
+					next;
+					select("But, I dont have enough Zeny.");
+					mes "[Metheus Sylphe]";
+					mes "The rules stated that everyone must made a donation, if they want to read the book...";
+					next;
+					mes "[Metheus Sylphe]";
+					mes "I guess sometime when life is difficult, we may not follow with the principle?";
+					next;
+					if (select("I must follow the principle!", "I'll do anything.") == 2) {
+						mes "[Metheus Sylphe]";
+						mes "I can feel the passion within you, but I couldn't let you enter for free.";
+						next;
+						mes "[Metheus Sylphe]";
+						mes "So help me resolve the problems happen in the Schweicherbil Magic Academy's library.";
+						mes "I'll help you to read the book of Ymir.";
+						do {
+							next;
+							switch(select("What happened?", "Where does it escaped?", "I'll find it.", "Ignore")) {
+								case 1:
+									mes "[Metheus Sylphe]";
+									mes "This is really a headache.";
+									mes "The library consists of a lot undisclosed books.";
+									next;
+									mes "[Metheus Sylphe]";
+									mes "For example, ^0055FFRideword Hat^000000 that would attack humans can't be disclose to the public.";
+									next;
+									mes "[Metheus Sylphe]";
+									mes "But not long ago, part of the undisclosed books escaped from the library...";
+									next;
+									mes "[Metheus Sylphe]";
+									mes "Although the library guards are currently searching for it, but failed due to lack of manpower.";
+									break;
+								case 2:
+									mes "[Metheus Sylphe]";
+									mes "We're still trying to figure out where it escaped to.";
+									mes "I'll inform you when the library guards have any news for it.";
+									break;
+								case 3:
+									mes "[Metheus Sylphe]";
+									mes "Do you understand the problem now?";
+									mes "Or you know other problems?";
+									next;
+									if (select("Sorry.", "Worry about gifts for girlfriend?", "Worry about the lunch?", "Worry about the escaped books?") == 4) {
```

Comment:
Text need to be added for each of the other cases

Reply:
updated